Services/Interfaces/web_interface/controllers/home.py

```diff
@@ -25,17 +25,13 @@
 @web_interface.server_instance.route("/home")
 @login.login_required_when_activated
 def home():
-    first_display = False
-    if flask.request.args:
-        accepted = flask.request.args.get("accept_terms") == "True"
-        models.accept_terms(accepted)
-        first_display = True
+    display_tutorial = str(flask.request.args.get("display_tutorial", False)) == "True"
```

Comment:
Do you think we could use something like [this](https://github.com/usablica/intro.js) for the tutorial ?